index.bs

```diff
@@ -2671,6 +2687,70 @@ credential. It is intended primarily for [RPS] that wish to tightly control the
     </pre>
 
 
+# IANA Considerations # {#IANA}
+
+## WebAuthn Attestation Format Identifier Registrations ## (#AttFmtReg)
+
+This section registers the attestation formats defined in Section [[#defined-attestation-formats]] in the
+IANA "WebAuthn Attestation Format Identifier" registry established by [[WebAuthn-Registries]].
+
+- WebAuthn Attestation Format Identifier: packed
+- Description: The "packed" attestation format is a WebAuthn-optimized format for attestation data. It uses a very compact but still extensible encoding method. This format is implementable by authenticators with limited resources (e.g., secure elements). 
+- Specification Document: Section [[#packed-attestation]] of this specification
+
+- WebAuthn Attestation Format Identifier: tpm
+- Description: The TPM attestation format returns an attestation statement in the same format as the packed attestation format, although the the rawData and signature fields are computed differently. 
+- Specification Document: Section [[#tpm-attestation]] of this specification
+
+- WebAuthn Attestation Format Identifier: android-key
+- Description: Platform-provided authenticators based on Android versions "N", and later, may provide this proprietary "hardware attestation" statement. 
+- Specification Document: Section [[#android-key-attestation]] of this specification
+
+- WebAuthn Attestation Format Identifier: android-safetynet
+- Description: Android-based, platform-provided authenticators may produce an attestation statement based on the Android SafetyNet API.
+- Specification Document: Section [[#android-safetynet-attestation]] of this specification
+
+- WebAuthn Attestation Format Identifier: fido-u2f
+- Description: Used with FIDO U2F authenticators
+- Specification Document: Section [[#fido-u2f-attestation]] of this specification
+
+## WebAuthn Extension Identifier Registrations ## (#ExtensionReg)
```

Comment:
overall comment wrt extension definitions in down in the "defined extensions" section (which is not (yet) modified by this PR):  each extn dfn should clearly note whether it is a "registration extension" or "authentication extension" (or both) -- I've denoted them as such in comments below.

Also, it would be good if in the "weauthn extensions" section (above) that the terms "registration extension" or "authentication extension" could be given the appropriate `<dfn>...</dfn>` markup, and then usage instances of the terms ought to get the `[=...=]` markup.